packages/pytest-simcore/src/pytest_simcore/simcore_services.py

```diff
@@ -135,7 +135,7 @@ def services_endpoint(
 
 
 @pytest.fixture(scope="module")
-async def simcore_services_ready(
+def simcore_services_ready(

```

Comment:
@GitHK because, as stated in the PR, we can only use **async fixtures with a function scope, otherwise (i.e. module or session) they must be synchronous**. This is a "reasonable" limitation provided the complex situations we found when mixing loops at different scopes :-)

Reply:
read the PR description ;)
async def fixtures are auto-detected by pytest-aiohttp

Reply:
Nice, I've missed that part. I got it, no more module or session scoped async fixtures, only function wise async.